### PR TITLE
Change Dart 1.X to Dart 1 text

### PR DESCRIPTION
**Pull request by joshvocal**
Staged: https://jv-dart-dev-1.firebaseapp.com/guides/language/sound-dart

I searched for all instances of Dart 1.X and replaced them.

Fixes: #927

### CODE REVIEW

**Comment by kwalrath** [comment]
I should check on this... The 1.X was intentional at the time, but maybe we don't want to do that any more.

File: src/_includes/async-await-2.0.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{site.alert.version-note}}
```
[the remark this answers is not in the document]
Comment (joshvocal) [reply src/_includes/async-await-2.0.md]: Just this version note in the markdown file or all version notes across the site?

[the remark this answers is not in the document]
Comment (kwalrath) [reply src/_includes/async-await-2.0.md]: Let's just delete this file. I just did a search, and it looks like nothing refers to it. (Feel free to check my work!)

I've filed #2174 to remind me to check other version-notes.

File: examples/strong/lib/strong_analysis.dart [hunk 2]
```diff
@@ -7,7 +7,7 @@ void _miscDeclAnalyzedButNotTested() {
   {
     // #docregion dart-2-note
     var i = 1;
-    // i is dynamic in Dart 1.x
```
[the remark this answers is not in the document]
Comment (joshvocal) [reply examples/strong/lib/strong_analysis.dart]: Sure, I'll make the changes to revert "Dart 1" -> "Dart1.X" and the note.

**Comment by kwalrath** [comment]
Thank you! I'll figure out why the build failed.

**Comment by kwalrath** [comment]
This build works for me locally without any changes, so I'm going to merge.

**Comment by kwalrath** [comment]
> This build works for me locally without any changes, so I'm going to merge.

Famous last words. The build is broken, but I'm working on it...

**Pull request merged**